crates/bdk/src/wallet/coin_selection.rs

```diff
@@ -749,9 +747,9 @@ mod test {
     use rand::seq::SliceRandom;
     use rand::{Rng, RngCore, SeedableRng};
 
-    // n. of items on witness (1WU) + signature len (1WU) + signature and sighash (72WU)
-    // + pubkey len (1WU) + pubkey (33WU) + script sig len (1 byte, 4WU)
-    const P2WPKH_SATISFACTION_SIZE: usize = 1 + 1 + 72 + 1 + 33 + 4;
+    // n. of items on witness (1WU) signature and sighash (72WU)
+    // + pubkey len (1WU) + pubkey (33WU)
+    const P2WPKH_SATISFACTION_SIZE: usize = 1 + 72 + 1 + 33;
```

Comment:
I think our accounting is right but just for clarity I would keep signature len in the above comment and drop witness length varint - which should already be included in [segwit_weight](https://docs.rs/bitcoin/0.30.2/bitcoin/blockdata/transaction/struct.TxIn.html#method.segwit_weight).

```suggestion
    // signature len (1WU) + signature and sighash (72WU)
    // + pubkey len (1WU) + pubkey (33WU)
    const P2WPKH_SATISFACTION_SIZE: usize = 1 + 72 + 1 + 33;
```

Reply:
I like this. I will add the commit and then squash it later